### PR TITLE
doc(website): Add MotherDuck how-to guide

### PR DESCRIPTION
**Pull request by hermanschaaf**
Adds a how-to guide for syncing data to MotherDuck

### CODE REVIEW

File: website/pages/how-to-guides/moving-data-from-postgres-to-motherduck.mdx [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+---
+
+import { HowToGuideHeader } from "../../components/HowToGuideHeader"
+import { Callout } from 'nextra-theme-docs';
+
+<HowToGuideHeader/>
+
+MotherDuck is a serverless analytics service powered by DuckDB. If you have data in a Postgres database that you'd like to analyze using MotherDuck, this tutorial will show you how to do this using CloudQuery, an open source ELT framework. We will cover:
+ 
+ 1. How to copy a Postgres database into MotherDuck as one-off batch operation, and
```
Comment (jsonpr) [inline website/pages/how-to-guides/moving-data-from-postgres-to-motherduck.mdx:17]:
```suggestion
 1. How to copy a Postgres database into MotherDuck as a one-off batch operation, and
```

File: website/pages/how-to-guides/moving-data-from-postgres-to-motherduck.mdx [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+ 2. How to continuously stream Postgres data from Postgres to MotherDuck using Postgres Change Data Capture (CDC).
```
Comment (jsonpr) [inline website/pages/how-to-guides/moving-data-from-postgres-to-motherduck.mdx:18]: Would be good to link to  https://www.cloudquery.io/blog/postgres-cdc-to-any-destination for Postgres CDC.

File: website/pages/how-to-guides/moving-data-from-postgres-to-motherduck.mdx [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+brew install cloudquery/tap/cloudquery
+```
+
+See the [CloudQuery Quickstart guide](/docs/quickstart) for installation instructions on other platforms. Please note that the DuckDB plugin (that we will use later) [does not support Windows](https://github.com/cloudquery/cloudquery/issues/8854) at this time.
+
+Once CloudQuery is installed, you should be able to invoke it from the command line:
+
+```shell
+$ cloudquery --version
+cloudquery version 3.5.0
```
Comment (erezrokah) [inline website/pages/how-to-guides/moving-data-from-postgres-to-motherduck.mdx:72]: Latest CLI version from Brew is 3.5.4
```suggestion
cloudquery version 3.5.4
```

Maybe the version placeholder will work too

**Review by erezrokah — APPROVED** [review]

**Comment by erezrokah** [comment]
Looks great 🚀

File: website/pages/how-to-guides/moving-data-from-postgres-to-motherduck.mdx (hunk 1 above)
Comment (hermanschaaf) [reply website/pages/how-to-guides/moving-data-from-postgres-to-motherduck.mdx]: Fixed, thanks!

File: website/pages/how-to-guides/moving-data-from-postgres-to-motherduck.mdx (hunk 2 above)
Comment (hermanschaaf) [reply website/pages/how-to-guides/moving-data-from-postgres-to-motherduck.mdx]: Nice, yeah - added a link 👍

**Pull request merged**